app/src/main/java/ch/epfl/sdp/blindly/UserMapActivity.kt

```diff
@@ -35,8 +54,56 @@ class UserMapActivity: AppCompatActivity(), OnMapReadyCallback, ActivityCompat.O
         val mapFragment = supportFragmentManager
                 .findFragmentById(R.id.map) as SupportMapFragment?
         mapFragment!!.getMapAsync(this)
+
+        // Cancel loading if we can't get the user id
+        val currentUserId = userHelper.getUserId() ?: return
+        val matchId = intent.extras?.getString(MATCH_ID) ?: "default_user"
```

Comment:
Sorry, due to my lack of thorough understanding of how this map works, I'll have to leave comments on trivial things...
I guess you could use a const String for the default_user, but feel free to ignore this comment........ x)

Reply:
In fact this seems to be same code I used for Chat, what it does is that when it creates an entry in the Realtime Database it names the other user as "default_user as a title, so instead of an error it just creates a trivial entry, I can't say it's the best way but it seems safer. For location, I don't know if it might cause problems if you try to fetch from "default_user" because it won't have any location. I'll let @koko-ng reply that. In terms of making it constant, certainly possible :D

Reply:
Hahaha no problem, it seems legit.